Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+#!groovy
```

Comment:
Why is this here? Jenkins can't access public github. We should be using GH Actions. Also, this should probably be under github.com/immediatemediaco.

Reply:
:doh: I'lll get some actions in. I put it here to avoid the pain that comes with the other repo which I understand isn't supported yet.

Reply:
Might be worth adding in here:
https://github.com/immediate-media/github-actions/tree/main/.github/workflows